src/services/search/SearchRequestMapper.res

```diff
@@ -0,0 +1,48 @@
+
+let mapSearchArtistToItemArtist = (artist: SearchResponse.artistItem) => {
+  let startImg: SearchResponse.image = {height: 999999, url: ""} 
```

Comment:
I would say we should use height 0 as a fallback

Reply:
You get only the url (that's what your model request). I use this height because otherwise the reduce function to get the min-height would not work.